docs/architecture/self-hosted/install-prophecy/installation-helm/installation-helm.mdx

```diff
@@ -79,9 +79,9 @@ Here are some configurations that are commonly changed from the default.
 
 By default, Prophecy will install its own managed Nginx ingress controller during the Helm installation. If you want to use your own Nginx ingress controller already installed on your Kubernetes cluster, you can do so if it supports external service exposure via a LoadBalancer. You'll have to set `global.disableIngressCreation` to true, and set `global.ingressController.class` to the appropriate name.
 
-#### Internal repository
+#### Private container registry
 
-During the Helm installation, docker images are pulled from Prophecy's public Google Container Registry `gcr.io/prophecy-share`. If you want to use your own internal container registry, you can specify it by setting the `global.repository` value during installation. Make sure to download the images from the public GCR first.
+During the Helm installation, docker images are pulled from Prophecy's public Google Container Registry `gcr.io/prophecy-share`. If you want to use your own internal container registry, you can specify it by setting the `global.repository` value during installation. Make sure to push all the images to your private registry first.
```

Comment:
You can replace 

 If you want to use your own internal container registry,
 
 with 
 
  If you want to use your own private container registry,